### PR TITLE
Support module install in container

### PR DESCRIPTION
**Pull request by dmarteau**
## Support for module installation in container

* Install composer
* Update docker/README.md for module installation
* Add helper for installing module: lizmap-install-module

* Ticket : #2286

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -20,15 +20,31 @@ The container deploys one Lizmap instance and may run php-fpm on the command lin
...
 
+### Note about configuration snippets.
+
+Configuration files may be splitted into snippets without tampering with the main configuration files. They must end with `.ini.php` and be valid php ini files.
+
+Configuration snippets are  searched in the following directories:
+
+* `$LIZMAP_CONFIG_INCLUDE/lizmapconfig.d`: all `*.ini.php` files in this directory will be imported to `lizmapConfig.ini.php`
+* `$LIZMAP_CONFIG_INCLUDE/localconfig.d`: all `*.ini.php` files in this directory will be imported to `localConfig.ini.php`.
+* `LIZMAP_PROFILES_INCLUDE`: Override for `$LIZMAP_CONFIG_INCLUDE/profiles.d`: all `*.ini.php` files in this directory will be imported to `localConfig.ini.php`.
```
Comment (laurentj) [inline docker/README.md:39]: `profiles.ini.php` I would say, no?

File: docker/README.md [hunk 2]
```diff
@@ -123,3 +139,34 @@ server {
...
 ```
 
+## Installing lizmap modules 
+
+### Install from archive
+
+- Mount `/www/lizmap/lizmap-modules/` at a location on the host or a from a named volume.
+- Get the archive of the lizmap module and extract it in the `/www/lizmap/lizmap-module/` or 
+  Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Add `<module_name>.access =2` to the `[modules]` section of `localconfig.ini.php` (which should be locatad in your `<install_dir>/lizmap/var/lizmap-config` directory
```
Comment (laurentj) [inline docker/README.md:149]: > Add `<module_name>.access =2`

Well. no, it depends of the module and how it works. the access value could be set into `var/config/index/config.ini.php` and/or `var/config/admin/config.ini.php`  etc... So just say to follow the documentation of the module to set right values into the `[modules]` section of the right configuration file.

File: docker/README.md [hunk 3]
```diff
@@ -123,3 +139,34 @@ server {
...
+- Mount `/www/lizmap/my-packages/` at a location on the host or a from a named volume.
+- From inside the container, install your module with the command `lizmap-install-modules <package-name>`
+- Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Restart the container or run `php /www/lizmap/install/installer.php` from inside the container
+
+### Install for module development
+
+- Mount `/www/lizmap/lizmap-modules/` at the location of your module sources.
+  Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Add `<module_name>.access =2` to the `[modules]` section of `localconfig.ini.php` (which should be locatad in your `<install_dir>/lizmap/var/lizmap-config` directory
```
Comment (laurentj) [inline docker/README.md:163]: same comment as above. Setting the access value depends of the module.

**Review by laurentj — CHANGES_REQUESTED** [review]

File: docker/README.md (hunk 2 above)
Comment (dmarteau) [reply docker/README.md:149]: Ok but that's weird that module activation configuration depends upon the module and is not unified...

File: docker/README.md (hunk 3 above)
Comment (dmarteau) [reply docker/README.md]: fixed

File: docker/README.md (hunk 2 above)
Comment (dmarteau) [reply docker/README.md]: fixed

File: docker/README.md (hunk 1 above)
Comment (dmarteau) [reply docker/README.md]: fixed

**Review by Gustry — APPROVED** [review]
Nice addition, LGTM

**Review by laurentj — APPROVED** [review]
r+

File: docker/README.md (hunk 2 above)
Comment (laurentj) [reply docker/README.md]: its weird, this why I made improvements and I created this my-packages directory, I created a plugin for Composer jelix/composer-module-setup to support [this `modules-autoconfig-access-16` section in composer.json](https://github.com/3liz/lizmap-wps-web-client-module/blob/master/composer.json#L22) of modules ;-)

**Pull request merged**